mix.exs

```diff
@@ -38,11 +38,14 @@ defmodule MLLP.MixProject do
       {:ranch, "~> 1.8.0"},
       {:elixir_hl7, "~> 0.6.0"},
       {:backoff, "~> 1.1.6"},
+      {:gen_state_machine, "~> 3.0"},
```

Comment:
This IMHO should go. Erlang is generally not a great thing to wrap, especially this kind of wrapping IMO which is very much subject to rot. If it was in stdlib sure, but yeah.